pkg/controller/certificates/sync.go

```diff
@@ -143,6 +184,21 @@ func (c *Controller) Sync(ctx context.Context, crt *v1alpha1.Certificate) (err e
 	// if we should being attempting to renew now, then trigger a renewal
 	if renewIn <= 0 {
 		err := c.renew(ctx, i, crtCopy)
+
+		// count result
+		result := resultSuccess
+		if err != nil {
+			result = resultFailure
+		}
+		c.metrics.IncCertificateRequestCount(
+			crt.Name,
+			crt.Namespace,
+			issuerName,
+			issuerKind,
+			issuerType,
+			operationRenew,
+			result)
```

Comment:
This too - `c.metrics.HandleCertificateRenew(crt, err)`

(the `operationRenew`, `resultFailure` and `resultSuccess` consts can move into the metrics package)